cloud/digital_ocean/digital_ocean_block_storage.py

```diff
@@ -0,0 +1,336 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+import json
+import time
+
+DOCUMENTATION = '''
+---
+module: digital_ocean_block_storage
+short_description: Create/destroy or attach/detach Block Storage volumes in DigitalOcean
+description:
+     - Create/destroy Block Storage volume in DigitalOcean, or attach/detach Block Storage volume to a droplet. 
+version_added: "2.2"
+author: "Harnek Sidhu"
+options:
+  command:
+    description:
+     - Which operation do you want to perform.
+    choices: ['create', 'attach']
+    required: true
+  state:
+    description:
+     - Indicate desired state of the target.
+    choices: ['present', 'absent']
+    required: true
+  api_token:
+    description:
+     - DigitalOcean api token.
+    required: true
+  block_size:
+    description:
+    - The size of the Block Storage volume in gigabytes. Required when command=create and state=present.
+  volume_name:
+    description:
+    - The name of the Block Storage volume.
+    required: true
+  description:
+    description:
+    - Description of the Block Storage volume.
+  region:
+    description:
+    - The slug of the region where your Block Storage volume should be located in.
+    required: true
+  droplet_id:
+    description:
+    - The droplet id you want to operate on. Required when command=attach.
+  timeout:
+    description:
+    - The timeout in seconds used for polling DigitalOcean's API.
+    default: 10
+
+notes:
+  - Two environment variables can be used, DO_API_KEY and DO_API_TOKEN.
+    They both refer to the v2 token.
+
+author:
+    - "Harnek Sidhu (github: harneksidhu)"
```

Comment:
Minor comment, but can you please change this to:
"Harnek Sidhu (@harneksidhu)"

In the future we may be parsing these lines